src/ansi-c/type2name.cpp

```diff
@@ -110,14 +112,21 @@ static std::string type2name(
   if(!type.source_location().get_function().empty())
     result+='l';
 
+  auto pointer_offset_bits_as_string = [&type, &ns]()
+  {
+    mp_integer bits = pointer_offset_bits(type, ns);
+    CHECK_RETURN(bits != -1);
+    return integer2string(bits);
+  };
```

Comment:
Is there a strong reason not to use a function at file scope?

Reply:
Not really. I can do that if you prefer.

Reply:
My reasons for preferring the file-scoped function: 1) Consistency: we don't do it elsewhere. 2) I'm not sure how well doxygen can cope with it. 3) The function calls do not make clear what is captured, thus making the code harder to read.

None of this is a blocker, but readability is a very important aspect.